nntrainer/layers/fc_layer.cpp

```diff
@@ -103,7 +103,7 @@ void FullyConnectedLayer::calcDerivative(RunLayerContext &context) {
   Tensor &derivative_ = context.getIncomingDerivative(SINGLE_INOUT_IDX);
   Tensor &ret_ = context.getOutgoingDerivative(SINGLE_INOUT_IDX);
 
-  derivative_.dot(weight, ret_, false, true);
+  ret_.dot_deriv_wrt_1(weight, derivative_, false, false);
```

Comment:
could you rename `ret_` to `d_input_` or sort for clarity?

Reply:
will do.